packaging/ubuntu/ubuntu_package_setup.sh

```diff
@@ -6,33 +6,44 @@
 # bash ubuntu_package_setup.sh [BRANCH] [WEB-UI-BRANCH]
 
 set -euo pipefail
+set -o xtrace
+
+SOURCE_DIR=$PWD
+
+SUDO=''
+if (( $EUID != 0 )); then
+    SUDO='sudo'
+fi
 
 BRANCH=${1:-master}
 WEB_UI_BRANCH=${2:-}
 
-BUILD_DIR="lbry-build-$(date +%Y%m%d-%H%M%S)"
+BUILD_DIR="$HOME/lbry-build-$(date +%Y%m%d-%H%M%S)"
 mkdir "$BUILD_DIR"
 cd "$BUILD_DIR"
 
 # get the required OS packages
-sudo add-apt-repository -y ppa:spotify-jyrki/dh-virtualenv
-sudo apt-get update
-sudo apt-get install -y build-essential git python-dev libffi-dev libssl-dev libgmp3-dev dh-virtualenv debhelper
+$SUDO apt-get -qq update
+$SUDO DEBIAN_FRONTEND=noninteractive apt-get -qq install -y --no-install-recommends software-properties-common
+$SUDO add-apt-repository -y ppa:spotify-jyrki/dh-virtualenv
+$SUDO apt-get -qq update
+$SUDO DEBIAN_FRONTEND=noninteractive apt-get -qq install -y --no-install-recommends build-essential git python-dev libffi-dev libssl-dev libgmp3-dev dh-virtualenv debhelper wget
 
 # need a modern version of pip (more modern than ubuntu default)
 wget https://bootstrap.pypa.io/get-pip.py
-sudo python get-pip.py
+$SUDO python get-pip.py
 rm get-pip.py
-sudo pip install make-deb
-
-# check out LBRY
-git clone https://github.com/lbryio/lbry.git --branch "$BRANCH"
+$SUDO pip install make-deb
 
 # build packages
+#
+# dpkg-buildpackage outputs its results into '..' so
+# we need to move lbry into the build directory 
+mv $SOURCE_DIR lbry
```

Comment:
it looks like this line is moving the current directory into lbry, not lbry into the current directory. is that what you intended? it seems to be the opposite of what the comment says